helpers/tools.ts

```diff
@@ -188,6 +188,35 @@ For better results, you can specify the region parameter to get results from a s
       },
     ],
   },
+  {
+    display: "Azure Code Interpreter",
+    name: "azure_code_interpreter.AzureCodeInterpreterToolSpec",
+    supportedFrameworks: ["fastapi"],
```

Comment:
Any reason this tool is limited to Python backed?

Reply:
in this PR, we just add it for Python only - next PR we add then your TS tool